### DISCUSSION TITLE
Will it work on this water meter?

### DISCUSSION THREAD

**Discussion by dxmone** [jomjol/AI-on-the-edge-device#1655] [General]
Hi, i have tried to search the disucssions and google if this solution will be able to take this kind of meter. But no luck in finding an answer so i will try here,

The image is quite bad but i want to be able to read the 5 digits from the meter. Is that possible in this solution. If so i will order the hardware needed.

![image](https://user-images.githubusercontent.com/36075893/208911639-bce4e48e-ddab-4890-a7d3-c1c4c64a61e5.png)

**Comment by caco3** [comment]
I do not see a reason why it should not work.
There are several LCD digits contained in the models, see https://jomjol.github.io/neural-network-digital-counter-readout/

**Reply by dxmone** [reply]
Thanks, i have placed an order on the HW and will try it out.

**Reply by jurasjo** [reply]
I use this project with this very model and I can confirm it works very well ;)
![Zrzut ekranu z 2022-12-21 16-55-34](https://user-images.githubusercontent.com/19259405/208934534-5c0d4813-77bc-4001-91a3-f04c9ada74f4.png)

I use this ESP32 Cam board:
https://www.partco.fi/fi/kehityskortit/esp/24113-esp32-esp32cam.html

**Reply by dxmone** [reply]
Perfect, will be nice to have water meter information in HA. Thanks for the replies.
